app-text/poppler/poppler23-23.09.0.recipe

```diff
@@ -517,14 +495,17 @@ INSTALL()
 
 	# GLIB
 	packageEntries "glib" \
+		$developLibDir/*glib* \
```

Comment:
devel libraries should not be added to runtime packages. Maybe split the devel package into multiple packages instead? Would that help?

Reply:
One of the reasons why I put draft on the PR, been side-tracked by the depending package checks.
Need to take a closer look how/if this can be done, don't want to add them back to the main _devel package as imho it shouldn't depend on glib/Qt. (we already got some issues (read the haikuports.cross issue the other day) with some there)

Reply:
I meant that you could add separate _glib_devel, _qt5_devel and _qt6_devel packages. There should be no problem with that (unlike debuginfo, of which there can be only one, but that isn't the problem here).

OpenSUSE does that too: https://build.opensuse.org/projects/openSUSE:Factory/packages/poppler/files/poppler.spec?expand=1

Reply:
Already on that :)

Reply:
> Already on that :) Was doing that before I "had" to reboot to Windows ;)
> 
> ```
> PROVIDES_qt5_devel="
> 	poppler23${secondaryArchSuffix}qt5_devel = $portVersion
> 	devel:libpoppler_qt5$secondaryArchSuffix = $libQt5VersionCompat
> 	"
> REQUIRES_qt5_devel="
> 	poppler23$secondaryArchSuffix == $portVersion
> 	"
> ```
> 
> WIP though, needs checking once done.

There is at least a typo: a missing `_` between `${secondaryArchSuffix}` and `qt5`, so `poppler23${secondaryArchSuffix}_qt5_devel`.

Also, it needs to be checked if the specific devel packages need to depend on the main one, so maybe add `poppler23${secondaryArchSuffix}_devel` to `REQUIRES_qt5_devel`.

Reply:
You were too fast, was working on that, pushed changes in regards to this just now. :)

Reply:
`poppler23$secondaryArchSuffix == $portVersion` for the sub _devel packages should be inherited by the main _devel package.